tools/validate_configs/test_configs/htcondor-pool.yaml

```diff
@@ -0,0 +1,117 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+blueprint_name: htcondor-pool
+
+vars:
+  project_id:  ## Set GCP Project ID Here ##
+  deployment_name: htcondor-001
+  region: us-central1
+  zone: us-central1-c
+  image_family: htcondor-9x
+  network_name: htcondor-pool
+  subnetwork_name: htcondor-pool-usc1
+
+deployment_groups:
+- group: htcondor-env
+  modules:
+  - source: modules/network/vpc
+    kind: terraform
+    id: network1
+    outputs:
+    - network_name
+  - source: community/modules/scripts/htcondor-install
+    kind: terraform
+    id: htcondor_install
+  - source: community/modules/project/service-enablement
+    kind: terraform
+    id: htcondor_services
+    use:
+    - htcondor_install
+  - source: modules/scripts/startup-script
+    kind: terraform
+    id: htcondor_install_scripts
+    settings:
+      runners:
+      - type: shell
+        source: modules/startup-script/examples/install_ansible.sh
+        destination: install_ansible.sh
+      - $(htcondor_install.install_htcondor_runner)
+    outputs:
+    - startup_script
+- group: packer
+  modules:
+  - source: modules/packer/custom-image
```

Comment:
My understanding is that this will use the default hpc image as a starting point and then will spit out an image named htcondor-9x after build. I do not like the implicit passing of image_family to this module and would prefer we do it similarly to how it is done in image-builder.yaml, by explicitly passing the output image name (leaving input as implicit default is fine though).

Reply:
`image-builder.yaml` doesn't do precisely what you describe. The produced image name is never used (or output) anywhere. However, the example it does not rely on the invisible propagation of `image_family` in several locations as this does. I think what you are proposing is to name the global variable anything except `image_family` and explicitly pass it?

Reply:
To clarify _"by explicitly passing the output image name"_, I should have said:

> by explicitly passing the output image name **to the packer module**. 
---
> I think what you are proposing is to name the global variable anything except image_family and explicitly pass it?

Yes, exactly this.

Reply:
In this case consider toolkit developers the user. If it make it clearer to read for people debugging the test there is still value in that.